src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1726,10 +1726,11 @@ public function getReturnUrl(WC_Order $order)
 	    $return_url = $this->removeTrailingSlashAfterParamater( $return_url );
         $order_id = wooCommerceOrderId($order);
         $order_key = wooCommerceOrderKey($order);
-        $return_url = add_query_arg(array(
-                                        'order_id'       => $order_id,
-                                        'key'            => $order_key,
-                                    ), $return_url);
+        $return_url = $this->getStringWithArgs(
```

Comment:
I would rename the method to something which reflect you'll get an url string with the given arguments.
This will explain better which is the purpose of the method.
Also, why prefixing with `get` ?